OJP/OJP_Lines.xsd

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="http://www.vdv.de/ojp" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:siri="http://www.siri.org.uk/siri" xmlns:gml="http://www.opengis.net/gml/3.2" targetNamespace="http://www.vdv.de/ojp" elementFormDefault="qualified" attributeFormDefault="unqualified" id="OJP_siri_service">
+	<xs:annotation>
+		<xs:documentation>OJP/OJP_Lines.xsd - OJP Line information request and response</xs:documentation>
+	</xs:annotation>
+	<xs:import namespace="http://www.siri.org.uk/siri" schemaLocation="../siri/siri_model/siri_all-v2.0.xsd"/>
+	<xs:include schemaLocation="OJP_All.xsd"/>
+	<xs:annotation>
+		<xs:documentation>=============================== Line Information Service ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:annotation>
+		<xs:documentation>=============================== Line Information Response Group ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:group name="LineInformationResponseGroup">
+		<xs:annotation>
+			<xs:documentation>Line Information response group.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="LineResult" type="LineResultStructure" minOccurs="0" maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:group>
+	<xs:annotation>
+		<xs:documentation>=============================== LineInformation support functions ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:complexType name="LineResultStructure">
+		<xs:sequence>
+			<xs:element name="LineRef" type="siri:LineRefStructure">
+				<xs:annotation>
+					<xs:documentation>Reference to the LINE.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element ref="siri:PublishedLineName" minOccurs="0"/>
+			<xs:element name="DirectionRef" type="siri:DirectionRefStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>DIRECTION of LINE.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="LinkProjection" type="LinkProjectionStructure" minOccurs="0">
```

Comment:
Project is expected to be on something in TM (usually on the infrastructure roard/rail)
The concept of "main geometry" sounds blurry to me and does not always exist.
Is it expected to be oriented (event if there is no direction)?
"an each" should be "and each"

Reply:
@Aurige : What should I correct? You disagree with Projection? We have this already in the Legs. The typo will be corrected.

Reply:
A Projection should refer to what it projects on ... If you would refer to some OMS (or equivalent) features, that would be Ok.
But, if I understand well, what is described here is the Route geometry (or part of it) ... So RouteGeometry would be better for the naming.

And on the functional side, I'm stille puzzled by the possible definition of "main geometry": what would you use for the RER A in Paris ? https://www.paris-metro-map.info/paris-rer-a-map/

Reply:
@Aurige How would you model this in Transmodel or NeTEx? I guess it is not possible either as this is done per Journey. Lines like that should better not exist in my view. If we want to show it that way, then I would need to model this with a completly new structure which probably needs to be a full graph (because circularity is possible as well). 
A possibility would be to allow multiple routes (then RER A is modeled as 6 routes.) What do you think?

Reply:
@Aurige Or do you wish me to expand the service that it also provides the line as a representation of stops (and then really doing the projection)?

Reply:
> A possibility would be to allow multiple routes (then RER A is modeled as 6 routes.) What do you think?

yes that's the way to go (I agree that one should avoid such geometry... but it does exist ans is comunicated to the public... no way to change this now)

Reply:
In fact you want a simple OJP service for a structure that can be complex in NeTEx ... at some point you just need to refer the NeTEx data.

Reply:
ok. opted for the reduced version.